from-romaji.cabal

```diff
@@ -38,4 +39,6 @@ executable from-romaji-exe
     , random-extras
     , random-fu
     , safe
+    , random
```

Comment:
I think you've edited the cabal file manually.
This cabal file is generated from `package.yaml` because we use `stack` to compile the project. Please add these packages to `stack.yaml` and make sure `stack build` doesn't throw any warnings.